src/mplfinance/plotting.py

```diff
@@ -264,7 +264,7 @@ def plot( data, **kwargs ):
         collections = _construct_ohlc_collections(xdates, opens, highs, lows, closes,
                                                          marketcolors=style['marketcolors'] )
     elif ptype == 'renko':
-        collections, new_dates, volumes, brick_values = _construct_renko_collections(dates, highs, lows, volumes, config['renko_params'], closes,
+        collections, new_dates, volumes, brick_values, brick_size = _construct_renko_collections(dates, highs, lows, volumes, config['renko_params'], closes,
```

Comment:
_construct_renko_collections() in _utils.py is not returning `brick_size`.    other than that, all looks good.

Reply:
Oops, I'll fix that now